ui/src/components/view/InfoCard.vue

```diff
@@ -794,6 +796,12 @@ export default {
     await this.getIcons()
   },
   computed: {
+    tagsSupportingResourceTypes () {
+      return ['UserVm', 'Template', 'ISO', 'Volume', 'Snapshot', 'Backup', 'Network',
+        'LoadBalancer', 'PortForwardingRule', 'FirewallRule', 'SecurityGroup', 'SecurityGroupRule',
+        'PublicIpAddress', 'Project', 'Account', 'Vpc', 'NetworkACL', 'StaticRoute', 'VMSnapshot',
+        'RemoteAccessVpn', 'User', 'SnapshotPolicy', 'VpcOffering']
```

Comment:
@shwstppr Is this the complete resources list supporting tags? how about keeping unsupported resource types only?

Reply:
@sureshanaparti some of the resource types used in the UI are not even present in the backend, https://github.com/shapeblue/cloudstack/blob/4.16/ui/src/config/section/infra/primaryStorages.js#L42